docs/deployment/images.md

```diff
@@ -55,3 +55,31 @@ root@dokku:~# dokku tags:deploy node-js-app v0.9.0
        http://node-js-app.dokku.me
 
 ```
+
+## Add Image Directly from Docker Hub
+
+Alternatively you can pull an image from Docker Hub and deploy it using the tagging feature.
+
+1. Create dokku app as usual
+
+       ```
+       root@dokku:~# dokku apps:create test-app
+       ```
+
+2. Pull image from Docker Hub
+
+       ```
+       root@dokku:~# docker pull demo-repo/some-image:v12
+       ```
+
+3. Retag the image to match the created app
+
+       ```
+       root@dokku:~# docker tag demo-repo/some-image:v12 dokku/test-app:latest
```

Comment:
Somehow it has to have a "latest" tag if this is the first time an empty app is tagged this way.